apps/api/src/app/agents/usecases/create-agent/create-agent.usecase.ts

```
@@ -1,9 +1,22 @@
-import { BadRequestException, ConflictException, Injectable, UnprocessableEntityException } from '@nestjs/common';
-import { AnalyticsService, shortId, slugifyOrRandom } from '@novu/application-generic';
-import { AgentRepository } from '@novu/dal';
+import {
+  BadRequestException,
+  ConflictException,
+  Injectable,
+  Logger,
```

Biome (2.4.15)

[error] Please use the PinoLogger from `@novu/application-generic` instead

(lint/style/noRestrictedImports)